perception/autoware_probabilistic_occupancy_grid_map/CMakeLists.txt

```diff
@@ -4,34 +4,58 @@ project(autoware_probabilistic_occupancy_grid_map)
 find_package(autoware_cmake REQUIRED)
 autoware_package()
 
+find_package(CUDA REQUIRED)
 find_package(eigen3_cmake_module REQUIRED)
 find_package(Eigen3 REQUIRED)
 find_package(PCL REQUIRED)
 
 include_directories(
+  include
   SYSTEM
+    ${CUDA_INCLUDE_DIRS}
     ${EIGEN3_INCLUDE_DIR}
+    ${grid_map_ros_INCLUDE_DIRS}
+)
+
+if(NOT ${CUDA_FOUND})
+  message(WARNING "cuda was not found, so the autoware_probabilistic_occupancy_grid_map package will not be built.")
+  return()
+endif()
```

Comment:
Since it was failing on both I prioritized fixing the cuda version first. Thanks to your suggestion, and adding cuda as non required, the cpu version passes (i.e., does not do anything), but the cuda one is still failing :cry:

Reply:
Thank you. Then I start investigating the CUDA job error.